libafl/src/executors/forkserver.rs

```diff
@@ -560,11 +560,23 @@ impl<'a, SP> ForkserverExecutorBuilder<'a, SP> {
         };
 
         for item in &self.arguments {
+            // need special handling for @@
```

Comment:
I would propose to not add this to the builder but have something like this:
https://github.com/AFLplusplus/LibAFL/blob/9d38fff662f4647117dff841fb9e8b30bc7ae5fe/libafl/src/executors/command.rs#L265

Reply:
We could even potentially share some code here 🤔

Reply:
so you mean separating this code from here and put it into another function in `ForkserverExecutorBuilder`?

Reply:
ah, ok then users can choose if they want the AFL-style parsing or not